### PR TITLE
PICARD-2696: avoid deleting dropped files

### PR DESCRIPTION
**Pull request by zas**
# Summary

* This is a…
  * [x] Bug fix
  * [ ] Feature addition
  * [ ] Refactoring
  * [ ] Minor / simple change (like a typo)
  * [ ] Other
* **Describe this change in 1-2 sentences**:

# Problem

* JIRA ticket (_optional_): PICARD-XXX

# Solution

# Action

### CODE REVIEW

File: picard/ui/coverartbox.py [hunk 1]
```diff
@@ -136,6 +136,7 @@ def dragMoveEvent(event):
         event.acceptProposedAction()
 
     def dropEvent(self, event):
+        event.setDropAction(QtCore.Qt.DropAction.CopyAction)
```
Comment (phw) [inline picard/ui/coverartbox.py:139]: Does this work without breaking normal move actions inside Picard? My idea would have been to only do this if the drop originates from outside.

**Review by phw — COMMENTED** [review]
Should work. Only open question seey comment. I'll test this when I'm back on the next weekend.

File: picard/ui/coverartbox.py (hunk 1 above)
Comment (Sophist-UK) [reply picard/ui/coverartbox.py:139]: As I commented previously elsewhere, one `proposedAction` is do nothing or Ignore. IMO you should NOT change this to a CopyAction.

Comment (phw) [reply picard/ui/coverartbox.py:139]: Picard only accepts copy and move actions. There is nothing else at that part of code.

File: picard/ui/itemviews.py [hunk 2]
```diff
@@ -695,8 +695,8 @@ def mimeTypes(self):
         return ["text/uri-list", "application/picard.album-list"]
 
     def dragEnterEvent(self, event):
+        event.setDropAction(QtCore.Qt.DropAction.CopyAction)
```
Comment (phw) [inline picard/ui/itemviews.py:698]: I wonder why in the reported case `event.mimeData().hasUrls()` would be false. After all it is files being dropped. 

It all looks like the reported issue was supposed to be handled, but for some reason it does not work. Really needs deeper debugging.

**Comment by phw** [comment]
@zas It needed a bit more handling, see #2270.

The main culprit was that in our case the parent implementation of `dropEvent` always did do a "acceptProposedAction", so no matter how we changed the actual action it always accepted the move.

**Pull request closed without merging**